problems/arrayMethods.js

```diff
@@ -156,7 +243,11 @@ const arePositiveEvenAndNonZeroEnding = () => {};
  * @returns {string[]} All strings uppercase.
  */
 
-const stringsAndCaps = () => {};
+const stringsAndCaps = (items) => { 
+items.map((el)=>{
+  return el.toUpperCase()
```

Comment:
some items in this array are not strings so they don't have the .toUpperCase() method available. Think about how you can filter out the array first....